packages/core/EventManager.ts

```diff
@@ -423,6 +423,7 @@ export default class EventManager {
     booking: PartialBooking,
     newBookingId?: number
   ): Promise<Array<EventResult<NewCalendarEventType>>> {
+    newBookingId = undefined;
```

Comment:
I don't know; are you still working on this? Setting this to undefined makes all kinds of dead code - please draft if so 👍

Reply:
Oh that's just me experimenting. Thanks for catching